docs/csharp/programming-guide/types/how-to-convert-a-string-to-a-number.md

```diff
@@ -13,46 +13,40 @@ ms.assetid: 467b9979-86ee-4afd-b734-30299cda91e3
 ---
 # How to convert a string to a number (C# Programming Guide)
 
-You can convert a [string](../../language-reference/builtin-types/reference-types.md) to a number by calling the `Parse` or `TryParse` method found on the various numeric types (`int`, `long`, `double`, and so on), or by using methods in the <xref:System.Convert?displayProperty=nameWithType> class.  
+You convert a `string` to a number by calling the `Parse` or `TryParse` method found on the various numeric types (`int`, `long`, `double`, and so on), or by using methods in the <xref:System.Convert?displayProperty=nameWithType> class.
```

Comment:
"Method found on the various" vs "method found on various" removing the 'the' feels more cleaner. Just a suggestion though. 

Rest of it is great 💯  @BillWagner